roles/grafana-repo/files/grafana.repo

```diff
@@ -0,0 +1,9 @@
+[grafana]
+name=grafana
+baseurl=https://packagecloud.io/grafana/stable/el/6/$basearch
+repo_gpgcheck=1
```

Comment:
The `repo_gpgcheck` is probably the problematic part which brings the issues with the grafana repo...